Server/VikingVault.API/Controllers/AutomaticPaymentController.cs

```diff
@@ -44,6 +44,19 @@ public ActionResult<AutomaticPayment> Post([FromBody] AutomaticPaymentDTO automa
             }
         }
 
+        [HttpPut]
+        public ActionResult<AutomaticPayment> Put([FromBody] AutomaticPaymentDTO automaticPayment)
+        {
+            try
+            {
+                return Ok(_automaticPaymentService.EditAutomaticPayment(automaticPayment));
+            }
+            catch (AutomaticPaymentServiceException apse)
```

Comment:
apse?

Reply:
It's the first letter from each word of the exception classname